internal/tests/suite_kernel_test.go

```diff
@@ -117,6 +117,9 @@ func (k *kernelVerifiableEndpoint) VerifyConnection(conn *networkservice.Connect
 	namingConn.Mechanism = &networkservice.Mechanism{
 		Cls:  cls.LOCAL,
 		Type: kernel.MECHANISM,
+		Parameters: map[string]string{
+			kernel.InterfaceNameKey: kernelmechanism.GetNameFromConnection(namingConn),
+		},
```

Comment:
Why do we need to add this?

I thought that kernel.Server could set `kernel.InterfaceName`

Reply:
In docker tests we want to check interfaces before ping. Kernel interface name is taken from mechanism. But at the moment we have the connection from the client side. Therefore, we need to update mechanism parameters, as if we are on the server side, to get the right kernel interface name.